igcollect/logfile_values.py

```diff
@@ -170,12 +208,26 @@ def parse_args():
     return parser.parse_args()
 
 
+def extract_timestamp_value(field):
```

Comment:
This isn't really exclusive to timestamps is it ? This could be any string value enclosed in quotes.